packages/identity-service/src/routes/authentication.js

```diff
@@ -39,11 +44,26 @@ module.exports = function (app) {
             paranoid: false
           })
           if (!existingRecord) {
+            // default to null
+            let walletAddress = null
+            req.logger.info({ headers }, "request headers")
+            if (headers && headers[EncodedDataMessageHeader] && headers[EncodedDataSignatureHeader]) {
+              const encodedDataMessage = headers[EncodedDataMessageHeader]
+              const encodedDataSignature = headers[EncodedDataSignatureHeader]
+              try {
+                walletAddress = sigUtil.recoverPersonalSignature({ data: encodedDataMessage, sig: encodedDataSignature })
```

Comment:
opposite of this call https://github.com/AudiusProject/audius-protocol/blob/main/packages/libs/src/services/web3Manager/Web3Manager.ts#L205